src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -127,6 +116,22 @@ Developer tools for Firefox for Android are provided through remote debugging me
 
 {% capture content %}
 
+## Execution model
+
+The process model on Android kills idle background processes to maintain device performance. Among the background processes Android can kill are extension processes. To account for this behavior, extensions should be designed to recover after they have been killed. [Event pages, or non-persistent background scripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Background_scripts), are the best mechanism to support this. If your extension uses persistent background script, [learn how to convert them to event pages](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Background_scripts#convert_to_non-persistent).
```

Comment:
This is probably acceptable enough of a description.  However the extension process is currently not running on android, so we may want to mention that it is comming...not sure.

Reply:
@mixedpuppy we probably don't want to include anything that's "coming" on the basis that there's always the possibility that there won't be the necessary flow of information to get the content updated. Unless this is going to be inaccurate for months, and to avoid diluting the message about using event pages, I'm inclined to suggest we just state this as is.

Reply:
@mixedpuppy could we either just omit the fact that we are technically not running the extensions in the extension child process at the moment (as Richard also pointed out, in the end it is something that will change under the hood, and not to the extension to choose, and so the extensions should be prepared anyway) or to mention something along the lines `... Android can kill are extension processes (on Firefox for Android versions there the extension child process is enabled)`?

Reply:
THat would be fine